public/js/group/create.js

```diff
@@ -1,15 +1,86 @@
+let meetingPlace = 'floor'
+const MEETING_PLACES = ['floor', 'link', 'other'] 
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function selectMeetingPlace(kind) {
+  meetingPlace = kind
+  const selectedButton = document.getElementById(`${kind}Btn`)
+  const selectedInputField = document.getElementById(`${kind}Input`)
+  const selectedWrapper = document.getElementById(`${kind}Div`)
+  selectedButton.classList.add('btn-meeting-selected')
+  selectedInputField.required = true
+  selectedWrapper.classList.remove('hidden')
+  const otherMeetingPlaces = MEETING_PLACES.filter(it => it !== kind)
+  otherMeetingPlaces
+    .map((it) => document.getElementById(`${it}Btn`))
+    .forEach((it) => {
+      it.classList.remove('btn-meeting-selected')
+    })
+  otherMeetingPlaces
+    .map((it) => document.getElementById(`${it}Input`))
+    .forEach((it) => {
+      it.required = false
+      it.value = ''
+    })
+  otherMeetingPlaces
+    .map((it) => document.getElementById(`${it}Div`))
+    .forEach((it) => it.classList.add('hidden'))
```

Comment:
Maybe this 3 forEaches could be put into one:

```js
MEETING_PLACES.filter(it => it !== kind).forEach(otherPlace => {
  document.getElementById(`${otherPlace}Btn`).classList.remove('btn-meeting-selected')
  const input = document.getElementById(`${otherPlace}Input`)
  input.required = false
  input.value = ''
  document.getElementById(`${otherPlace}Div`).classList.add('hidden')
})
```

Reply:
Reason: more compant code, can be fluently read along

Reply:
Yeah, it's way better and nicer, thank you :D
The first version of this function was a disaster compared to this